provider/ansible.py

```diff
@@ -132,17 +132,18 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # Install ansible depended packages that can't be installed
+        # automatically by pip when installing ansible
+        package_list = params.objects('package_list')
+        if package_list:
+            for package in package_list:
```

Comment:
```python
def package_install(pkg, session=None, timeout=PKG_MGR_TIMEOUT):
    """
    Try to install packages on system with package manager.

    :param pkg: package name or list of packages
    :param session: session Object
    :param timeout: timeout for install with session
    :return: True if all packages installed, False if any error
    """
```

So here only need a package list, not a `for` loop.

Reply:
Updated. Thanls.